lib/resolvers/PackageResolver.ts

```diff
@@ -105,6 +117,24 @@ export class PackageResolver {
     });
   }
 
+  /**
+   * Retrieves the latest Angular Material version remotely.
+   */
+  private static _retrieveLatestVersion(): Promise<string> {
+    return new Promise((resolve, reject) => {
+      request('http://material.angularjs.org/docs.json', (error, response, body) => {
```

Comment:
I went with it since we also depend on it for testing the different versions. I'll check if npm exposes the `view` command programatically.